lib/3scale_toolbox/configuration.rb

```diff
@@ -0,0 +1,45 @@
+require 'yaml/store'
+
+module ThreeScaleToolbox
+  class Configuration
+    ATTRIBUTES = %i[remotes].freeze
+    private_constant :ATTRIBUTES
+
+    def initialize
+      @store = YAML::Store.new(ThreeScaleToolbox.config_file)
```

Comment:
IMO this is wrong direction of dependencies. The file name of the store should be pushed here, not pulled from somewhere.